### PR TITLE
fixes in Grid view and updates for remix guide

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

**Comment by ryestew** [comment]
Questions
1. how are the logos generated and what is this logo link?

Text update - to be addressed on a PR onto this PR
1.  "title": "Remix Guide" - 
- if these are all videos then it should be:
    - Remix Videos
    - Remix Curated Videos

2. Description update:
Categorized video tutorials on using Remix and developing in Solidity. 

3. Should the logos link to the video maker's site?

File: libs/remix-ui/modal-dialog/src/lib/remix-ui-modal-dialog.tsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ export const ModalDialog = (props: ModalDialogProps) => {
   }
 
   const modalKeyEvent = (keyCode) => {
+    console.log("modal key", keyCode)
```
Comment (Aniket-Engg) [inline libs/remix-ui/modal-dialog/src/lib/remix-ui-modal-dialog.tsx:50]: remove log

File: apps/remix-ide/src/app/plugins/remixGuide.tsx [hunk 2]
```diff
@@ -24,38 +23,47 @@ export class RemixGuidePlugin extends ViewPlugin {
...
 
   async onActivation() {
     this.handleThemeChange()
     await this.call('tabs', 'focus', 'remixGuide')
     this.renderComponent()
     _paq.push(['trackEvent', 'plugin', 'activated', 'remixGuide'])
+    // Read the data
+    this.payload.data = Data
+    this.handleKeyDown = (event) => {
+      console.log("key down")
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app/plugins/remixGuide.tsx:52]: log

File: libs/remix-ui/modal-dialog/src/lib/remix-ui-modal-dialog.tsx (hunk 1 above)
Comment (LianaHus) [reply libs/remix-ui/modal-dialog/src/lib/remix-ui-modal-dialog.tsx]: i did already

File: apps/remix-ide/src/app/plugins/remixGuide.tsx (hunk 2 above)
Comment (LianaHus) [reply apps/remix-ide/src/app/plugins/remixGuide.tsx:50]: just pushed please refresh

**Comment by LianaHus** [comment]
> Questions
> 
> 1. how are the logos generated and what is this logo link?

you need to provide the video id

> 2. how to add new tags?
> 

either in the plugin or can make it part of json. 
> Text update - to be addressed on a PR onto this PR
> 
> 1. "title": "Remix Guide" -
> 
> * if these are all videos then it should be:
>   
>   * Remix Videos
>   * Remix Curated Videos
> 
kind of like guide more. videos sound too obvious. let's discuss
> 3. Description update:
>    Videos about Remix and developing in Solidity.

why solidity?
> 4. Should the logos link to the video maker's site?

Will do this. good idea

File: libs/remix-ui/grid-view/src/lib/remix-ui-grid-cell.tsx [hunk 3]
```diff
@@ -16,67 +16,97 @@ interface RemixUIGridCellProps {
...
   useEffect(() => {
     if (props.tagList) setAnyEnabled(props.tagList.some((key) => filterCon.keyValueMap[key]?.enabled))
-    else setAnyEnabled(filterCon.showUntagged)
+    else setAnyEnabled(filterCon?.keyValueMap['no tag']?.enabled)
+    if (filterCon.filter != '') setAnyEnabled(anyEnabled && props.title.toLowerCase().includes(filterCon.filter))
+    console.log("pin ", pinned)
+
   }, [filterCon, props.tagList])
 
+  /*const listenOnExpand = (key) => {
```
Comment (yann300) [inline libs/remix-ui/grid-view/src/lib/remix-ui-grid-cell.tsx:42]: you should probably remove if it's unused.

File: apps/remix-ide/src/app/plugins/remixGuide.tsx (hunk 2 above)
Comment (yann300) [reply apps/remix-ide/src/app/plugins/remixGuide.tsx:50]: the log is still tehre.

**Review by yann300 — APPROVED** [review]

File: libs/remix-ui/grid-view/src/lib/remix-ui-grid-cell.tsx (hunk 3 above)
Comment (LianaHus) [reply libs/remix-ui/grid-view/src/lib/remix-ui-grid-cell.tsx:42]: I will use this for next iteration. It's disabled but flags are there.

**Pull request merged**